gulpfile.js

```diff
@@ -36,9 +36,11 @@ gulp.task('clean', (done) => {
 });
 
 gulp.task('build', ['clean'], () => {
-  const tsResult = tsProject.src().pipe(typescript(tsProject));
-
-  return mergeStream(tsResult.dts, tsResult.js).pipe(gulp.dest('lib'));
+  let tsReporter = typescript.reporter.defaultReporter();
+  return mergeStream(
+             tsProject.src().pipe(tsProject(tsReporter)),
+             gulp.src(['src/**/*', '!src/**/*.ts']))
+      .pipe(gulp.dest('lib'));
```

Comment:
Yup! fwiw, this was adapted from `polymer-cli` where this had already been updated